packages/rocket_chat_api/windows/flutter/generated_plugin_registrant.cc

```diff
@@ -0,0 +1,11 @@
+//
+//  Generated file. Do not edit.
```

Comment:
@yatendra2001   please take a look at    https://github.com/flutter/flutter/blob/master/.gitignore

These generated files should not be committed.